node/src/actors/chain_manager/mod.rs

```diff
@@ -1834,6 +1845,29 @@ fn show_sync_progress(
     );
 }
 
+// TODO: handle recovery cases after reduction
+fn calculate_committee_size(
```

Comment:
Maybe we could change the name to `update_committee_size`, since the committee size is a consensus constant and you don`t need to calculate it, does it make sense?

Reply:
Hmm I agree that calculate might not be the best name for it. But the truth is that based on your last consolidated epoch and the default committee size you will always have to calculate it... Update sounds a bit like if we were gonna update the consensus constant for me. Any more ideas for the naming?